src/Nethermind/Nethermind.Db/Metrics.cs

```diff
@@ -122,6 +122,26 @@ public static class Metrics
         [Description("Indicator if StadeDb is being pruned.")]
         public static int StateDbPruning { get; set; }
 
+        [GaugeMetric]
+        [Description("Size of state DB in bytes")]
+        public static long StateDbSize { get; set; }
+
+        [GaugeMetric]
+        [Description("Size of receipts DB in bytes")]
+        public static long ReceiptsDbSize { get; set; }
+
+        [GaugeMetric]
+        [Description("Size of headers DB in bytes")]
+        public static long HeadersDbSize { get; set; }
+
+        [GaugeMetric]
+        [Description("Size of blocks DB in bytes")]
+        public static long BlocksDbSize { get; set; }
+
+        [GaugeMetric]
+        [Description("Size of all dbs")]
+        public static long DbSize { get; set; }
+
```

Comment:
In for example `Db.Rocks.Config.DbConfig` we also have `BlockInfosDb`, `PendngTxsDb`, `CodeDb`, `BloomDb`, `WitnessDb`, `CanoncalHashTrieDb`, `MetadataDb`. Why they are not monitored?

Reply:
I think grafana is payed per metric. So I added only most important ones

Reply:
Grafana is free. Please add all so that users could decide what to filter on the Prometheus level for example.